### ISSUE TITLE
[Legitimate Site Blocked] openseam.co

### ISSUE THREAD

**Issue by bodovoet** [MetaMask/eth-phishing-detect#12131]
https://openseam.co/

We make an IoT solution for textile factories. Some of our customers got blocked access to our company website. It seems to be triggered by the Metamask browser extension.

Attached is what our company website looks like, for reference. 

Please unblock! Can you also please explain how it came to be blocked? We have nothing to do with your world of crypto...

thank you!

Bodo Voet
CEO, OpenSeam

![screencapture-openseam-co-2023-03-27-12_36_14](https://user-images.githubusercontent.com/31516980/228048148-e2a9684f-6119-4666-b26c-8d215d6fdfce.png)

**Comment by bodovoet** [comment]
thank you @deshvin for the progress, though my do you have any indication when it will go through? It still shows as blocked. thanks!

**Comment by bodovoet** [comment]
@deshvin my site is still being blocked. When is the allowlist going through?!

**Comment by bodovoet** [comment]
@deshvin it seems that something must be wrong with your allowlist going through. I will submit a new ticket request to unblock my site, because this is taking too long~!

**Comment by deshvin** [comment]
apologies for the delay, there was an issue with the PR so it was not merged. I'm continuing to look into it.

**Comment by bodovoet** [comment]
heck yes!! thanks @deshvin!
